### DISCUSSION TITLE
Problem with "Parameter type is Context[Unknown]"

### DISCUSSION THREAD

**Discussion by Matieus** [Rapptz/discord.py#8250] [Q&A]
![Bez tytułu](https://user-images.githubusercontent.com/76398414/179809412-62e034d4-fe72-462f-9590-284d873b2bd7.png)
Hey, what's the problem with that annotation?
After the holiday, I started my project and I see these types of errors.
So I checked on a clean design and I can see the same errors. I'm using Pylance linter, discord.py 2.0.0a, Python 3.9.4.
How can I fix it? Only Pylance reports annotation errors. The bot works.

**Comment by imvaskel** [comment]
Context is now generic, meaning it expects a type argument of a subclass of `commands.Bot`. You will want to annotate it as `Context[MyBot]`.

**Reply by Matieus** [reply]
Will commands.Bot work too?
Because I store the bot instances in a variable

**Reply by Gobot1234** [reply]
Yes, but you'll lose type information (ie if you have some specific methods on your bot subclass you won't get intellisense for them). You might be better using [`if TYPE_CHECKING`](https://docs.python.org/3/library/typing.html#typing.TYPE_CHECKING)
